backend/pyproject.toml

```diff
@@ -0,0 +1,75 @@
+[tool.poetry]
+name = "cognita"
+version = "0.0.0"
+description = "Cognita"
+authors = ["Abhishek Choudhary <abhichoudhary06@gmail.com>"]
+readme = "README.md"
```

Comment:
This does not exist relative to this file
We want to move pyproject.toml to root